src/payment/strategies/adyenv2/adyenv2-initialize-options.ts

```diff
@@ -16,9 +16,9 @@ export default interface AdyenV2PaymentInitializeOptions {
     containerId: string;
 
     /**
-     * The location to insert the Adyen 3DS V2 component.
+     * The location to insert the additional action component.
      */
-    threeDS2ContainerId: string;
+    additionalActionContainerId: string;
```

Comment:
I'm sorry but you can't rename fields once it's merged to master. You can only make additive changes that are backward compatible. One way to do this is to read from both properties, but mark the first one as deprecated (so both 3DS and Bank Transfer will use the same container). Another way is introduce a second field called `bankTransferContainerId`, which is only used when bank transfer is required. The first approach might be an easy solution as the client doesn't need to do anything and it can be compatible with other action types such as QR code etc..., but if in the future, 3DS requires a different treatment than bank transfer, you won't be able to do it. The second approach gives you more control over what the client needs to provide in order to support 3DS and bank transfer. But in the future, if we need to support more actions, we'll need to add more fields and ask the clients to update. Judging on the Adyen documentation, it seems the first approach might align better with what they offer https://docs.adyen.com/checkout/components-web#step-4-additional-front-end